docs/tutorial/part05.rst

```diff
@@ -2,5 +2,126 @@
 How to create the API.
 ======================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+Why the API structure?
+======================
+
+We decided to use APIs to fetch the data to make queries more effective, as sometimes queries get
+too big. We decided it was better to fetch the data in little chunks rather than one big request.
+The API structure allows us to do so and to have a single page application. We will be returning
+JSON to the calls since it is easily usable on the frotend side.
+
+Implementing Django REST framework for API
+==========================================
+
+In this project, we are using the rest_framework_ library because it has some simple non-limiting
+ways to implement an API structure. If you desire you can use any of your preference, but the
+examples presented ahead will be based on the behavior of this specific library.
+
+    .. _rest_framework: http://www.django-rest-framework.org/
+
+You should have the library already installed since it is in the requirements file that we provided
+for the project. If not just run::
+
+    pip install djangorestframework
+
+On your console and restart your Django server.
+
+Create the views / endpoints
+============================
+
+We are going to need a couple different endpoints to make our application work, but for now, we will
+focus on one to explain the process of making them, we are going to be building the
+``GetNodesData / nodes`` that will serve us to fetch the search matches nodes data. We'll be using a
+type of class-based view called ``APIView`` from the framework to make them, this will allow us to
+have an endpoint rather than a page view.
+
+Clean your ``fetch_api/views.py`` file and add the following code::
+
+    from rest_framework.views import APIView
+    from rest_framework.response import Response
+    from rest_framework import status
+
+    class GetNodesData(APIView):
+        def get(self, request):
+            return Response('Temporary Data', status=status.HTTP_200_OK)
+
+Now create this file ``fetch_api/urls.py`` and fill it with this::

```

Comment:
the file ``fetch_api/urls.py`` is created in part 3.

 You can see it here: https://github.com/neo4j-examples/paradise-papers-django/pull/51/files#diff-c1e0a5d898826045a3e281cad742e750R25